hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconIncrementalContainerReportHandler.java

```diff
@@ -46,7 +46,7 @@
       ReconIncrementalContainerReportHandler.class);
 
   public ReconIncrementalContainerReportHandler(NodeManager nodeManager,
-             ContainerManager containerManager, SCMContext scmContext) {
+                    ContainerManager containerManager, SCMContext scmContext) {
```

Comment:
I think we should remove this unnecessary change in the class

Reply:
Removed thanks !!